source/ui/dataman/AMSpectrumAndPeriodicTableView.h

```diff
@@ -156,6 +156,8 @@ protected slots:
 
 	/// The export button.
 	QPushButton *exportButton_;
+	///
```

Comment:
Why is this function buried deep within the protected members when it is supposed to be a slot?